pkg/selinux/selinux.go

```diff
@@ -269,20 +268,37 @@ func SelinuxGetEnforceMode() int {
 	return Disabled
 }
 
+func mcsPath(mcs string) string {
+	filename := fmt.Sprintf("/run/rkt/%s", mcs)
+	return filename
```

Comment:
Packages in the `pkg/` directory should be pretty independent of rkt and be able to be included in other projects. For example, acbuild includes [several github.com/coreos/rkt/pkg/* packages](https://github.com/appc/acbuild/blob/master/Godeps/Godeps.json#L47).

I wonder if the functions here could be methods on an object instantiated with the "rkt" or "/run/rkt" parameter. But I don't know if it worth the change. @krnowak?

Also, we have already `/run/rkt/metadata-svc.sock`. The name does not seem to conflict with `mcs`, but can we put the mcs in a subdirectory just in case we add more things in `/run/rkt` later?